rules/autoconsent/pubtech.json

```diff
@@ -1,30 +1,30 @@
 {
-    "name": "pubtech",
-    "prehideSelectors": ["#pubtech-cmp"],
-    "detectCmp": [
-      { 
-        "exists": "#pubtech-cmp" 
-      }
-    ],
-    "detectPopup": [
-      { 
-        "visible": "#pubtech-cmp" 
-      }
-    ],
-    "optIn": [
-      { 
-        "click": "#pubtech-cmp #pt-accept-all" 
-      }
-    ],
-    "optOut": [
-      {
-        "click": "#pubtech-cmp #pt-close:first-child" 
-      }
-    ],
-    "test": [
-      {
-        "eval": "document.cookie.includes('euconsent-v2=CPeDpcAPeDpcAFgADAENCaCgAAAAAAAAABCYAAAAAAAA.YAAAAAAAAAAA') && !document.cookie.includes('IDE')"
-      }
-    ]
-  }
-  
\ No newline at end of file
+  "name": "pubtech",
+  "prehideSelectors": ["#pubtech-cmp"],
+  "detectCmp": [
+    { 
+      "exists": "#pubtech-cmp" 
+    }
+  ],
+  "detectPopup": [
+    { 
+      "visible": "#pubtech-cmp #pt-accept-all, #pt-actions"
+    }
+  ],
+  "optIn": [
+    {
+      "click": "#pubtech-cmp #pt-actions button:nth-of-type(2)"
```

Comment:
hm, what if they change the order of buttons?

Reply:
This had me scratch my head because:
- Yes they could switch the order and I'm aware that since it's the "opt-in" piece of the rules we may not know that we aren't capturing this click.
- As this site is mostly autogenerating its selectors, I can't seem to find a good combo of them that will reliably grab that click. Especially since there's two types of consent banners: "https://www.fattoincasadabenedetta.it/" is the most common one. "https://www.lefonti.tv/" is the exception so far <- they only seem to have this stable selector in common for that element
- An idea is to capture the button's wording (i.e. "Accept All") but this CMP seems to mostly cover Italian websites, and from the US they use English speaking words. Wording does not seem to be a reliable option

If there's no good option, shall we park that CMP under the autogenerated selectors project?
Did you have something in mind?

Reply:
It seems that in most cases "#pt-accept-all" would work. Let's do a simple conditional here? if it exists, click it, if it doesn't exist, fall back to nth-of-type